@factor/@themes/theme-ultra/src/page-home.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div id="homeContainerID" class="homeContainer">
+    <div class="homeQuotesContainer">
```

Comment:
classnames should be kebab cased in core-factor. Style guide is needed on our part

Reply:
This change will require a bit more time but I will try to add it in my next PR, thanks!